tests/Stubs/AbstractAuthenticationService.stub

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace TYPO3\CMS\Core\Authentication;
+
+/**
+ * This file can be removed once https://review.typo3.org/c/Packages/TYPO3.CMS/+/77385 is released.
```

Comment:
This patch was merged yesterday. The next TYPO3 v12 release is scheduled for 2023-02-07.